tempodb/encoding/common/types.go

```diff
@@ -1,7 +1,44 @@
 package common
 
+import (
+	"hash"
+	"hash/fnv"
+)
+
 // This file contains types that need to be referenced by both the ./encoding and ./encoding/vX packages.
 // It primarily exists here to break dependency loops.
 
 // ID in TempoDB
 type ID []byte
+
+// IDMap is a helper for recording and checking for IDs. Not safe for concurrent use.
```

Comment:
not seeing this used anywhere. remove?

Reply:
It's used in parquet wal block FindTraceByID, to only read from pages containing the trace. The map is maintained on append/flush and rebuilt on rescan.

Reply:
whoops. didn't expand that diff :P